### PR TITLE
Introduce a CLI for configuration registry

### PR DESCRIPTION
**Pull request by kyessenov**
Example:  
```
> cat example
dst_service_name: test
> cat example | bazel-bin/cmd/manager/manager config put route-rule  test  -c ~/.kube/config 
> manager config list route-rule -c ~/.kube/config 
Kind route-rule, Name: test, Namespace: default
======
dstservicename: test
routerule: null


```

### CODE REVIEW

**Comment by rshriram** [comment]
dst_Service_name has changed to destination

File: model/registry.go [hunk 1]
```diff
@@ -65,6 +70,44 @@ type ProtoSchema struct {
...
+	pb := reflect.New(pbt.Elem()).Interface().(proto.Message)
+	err = jsonpb.UnmarshalString(string(str), pb)
+	if err != nil {
+		return nil, err
+	}
+
+	return pb, nil
+}
+
 const (
```
Comment (rshriram) [inline model/registry.go]: Move these to to converter.go perhaps?

File: platform/kube/client.go [hunk 2]
```diff
@@ -232,7 +232,8 @@ func (cl *Client) Get(key model.Key) (proto.Message, bool) {
 		return nil, false
 	}
 
-	out, err := mapToProto(cl.mapping[key.Kind].MessageName, config.Spec)
+	kind := cl.mapping[key.Kind]
+	out, err := kind.FromJSONMap(config.Spec)
 	if err != nil {
```
Comment (rshriram) [inline platform/kube/client.go]: I think we can definitely move those two to conversion.go

Comment (kyessenov) [inline platform/kube/client.go]: We need protobuf specific conversion stuff in `model/conversion.go`. `kube/conversion.go` is for k8s

File: model/registry.go (hunk 1 above)
Comment (kyessenov) [inline model/registry.go]: Yeah

**Comment by kyessenov** [comment]
These examples should work:
```
bazel run //cmd/manager -- config list route-rule
echo destination: test | bazel-bin/cmd/manager/manager config put route-rule test
bazel run //cmd/manager -- config get route-rule test
```

**Pull request merged**